packages/geoprocessing/src/components/table/ClassTable.stories.tsx

```diff
@@ -367,6 +368,36 @@ export const categoricalData = () => {
   );
 };
 
+export const missingData = () => {
+  return (
+    <Translator>
+      <ClassTable
+        rows={missingAndZeroMetrics}
+        metricGroup={categoricalMetricGroup}
+        objective={categoricalSingleObjective}
```

Comment:
This story doesn't yet properly expose the tables handling of missing metrics, specifically making them 0 and showing an info Tooltip.  cc @avmey can you help suggest what input will produce this?

Reply:
I thought both a missing or a zero value metric might do it.

```json
// Missing MadracisReef
export const missingAndZeroMetrics: Metric[] = createMetrics([
  {
    classId: "Bays and Coast",
    value: 0,
    metricId: categoricalMetricGroup.metricId,
  },
]);
```

Reply:
Currently the tooltips appear when a metric has a NaN value, which is set when toPercentMetric() is called with a 0 denominator. This is because have a value of 0 could also mean a user didn't overlap with the feature, as opposed to the feature not existing in that geography.